piptools/scripts/compile.py

```diff
@@ -326,6 +326,23 @@ def _determine_linesep(
     help="Do not read any config file.",
     is_eager=True,
 )
+@click.option(
+    "-c",
+    "--constraint",
+    type=click.Path(
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        readable=True,
+        allow_dash=False,
+        path_type=str,
+    ),
+    multiple=True,
+    help=(
+        "Constrain versions using the given constraints file. "
+        "This option can be used multiple times."
```

Comment:
All the other `multiple=True` options are using the exact phrase "may be used more than once" 🤷🏼

Reply:
Thanks! Addressed in d5e13f73201aa653835dd35c347b3f82011965fd.